src/components/molecules/loginModal.jsx

```diff
@@ -11,14 +11,12 @@ function LoginModal({ isOpen, handleClose }) {
 
     const handleSubmit = (e) => {
         e.preventDefault();
-        console.log(username);
-        console.log(password);
         login(username, password);
         handleClose();
     }
 
     return (
-        (isOpen && <div className='absolute top-0 left-0 flex justify-center items-center h-full w-full bg-black bg-opacity-25 z-30'>
+        (isOpen && <div className='absolute top-0 left-0 flex justify-center items-center h-[100vh] w-full bg-black bg-opacity-25 z-30'>
```

Comment:
You don’t necessarily have to change it, but I thought changing absolute to fixed would provide a better user experience by keeping the modal fixed in place relative to scrolling